test/suite/test_prepare04.py

```diff
@@ -106,14 +106,17 @@ def test_prepare_conflict(self):
         s_other.begin_transaction(self.txn_config + self.ignore_config)
         c_other.set_key(1)
         if self.ignore == False and self.after_ts == True:
-            self.assertRaises(wiredtiger.WiredTigerError, lambda:c_other.search())
+            # WT_PREPARE_CONFLICT error returned when a read encounters a prepared transaction update
```

Comment:
I would make this comment more general like:
`# Make sure we get the expected prepare conflict message.`

Reply:
updated according to your comment.